includes/apple-exporter/builders/class-components.php

```diff
@@ -526,6 +526,10 @@ private function get_components_from_node( $node ) {
 	 */
 	private function get_image_full_size_url( $url ) {
 
+		if(null === $url) {
```

Comment:
```suggestion
		if( empty( $url ) ) {
```

Reply:
Additionally, this is going to fail phpcs checks because there needs to be a space after the `if`. You should be running `composer phpcs` locally and ensuring it is clean before creating a PR, and should ensure that all of the automated checks pass before putting it in for code review.

Reply:
Noted, I didn't want to run phpcbf yet because so many lines of code were going to be impacted, and I wanted to get early feedback, but will do before I finalize this PR. Thank you.